### ISSUE TITLE
light theme join server button

### ISSUE THREAD

**Issue by Alexandreblnqro** [revoltchat/revite#335]
I'm using light theme, and when I joined the server theme I just get a black button on the invitation page. I don't think it's normal so I prefer to report that.
![image](https://user-images.githubusercontent.com/71755937/136763514-211a923d-ed38-4849-8296-68444c625880.png)

**Comment by Steveplays28** [comment]
If you disable the background-color property in the CSS it fixes the issue, if the image below is how it's supposed to be... seems like a bug?
No clue how to fix it though.

![image](https://user-images.githubusercontent.com/62797992/141356343-7a8711b3-3d0f-44a9-a7f7-c1dc2dae61b2.png)

**Comment by Steveplays28** [comment]
Nice thank you!
